src/lib/components/consignments/components/boolean_button.tsx

```diff
@@ -0,0 +1,94 @@
+import * as React from "react"
```

Comment:
Him, maybe an onchange is just better overall, it can provide the new state then